modules/administration/pages/hostname-rename.adoc

```diff
@@ -0,0 +1,36 @@
+[[hostname-rename]]
+= Rename {productname} Server
+
+It is possible to rename a {productname} Server.
+Begin by changing the network settings of the server locally and remotely.
+Then use the [command]``spacewalk-hostname-rename`` script to update the settings in the PostgreSQL database and the internal structures of {productname}.
+
+The [command]``spacewalk-hostname-rename`` script is part of the [package]``spacewalk-utils`` package.
+
+The only mandatory parameter for the script is the newly configured IP address of the {productname} Server.
+
+
+
+.Procedure: Renaming {productname} Server
+. Change the network settings of the server on the system level locally and remotely at the DNS server.
+Provide configuration settings for reverse name resolution, too.
+Changing network settings is done in the same way as with renaming any other system.
+. Restart your network configuration to make the {productname} Server uses the new hostname.
+. Reboot the {productname} Server to ensure the hostname has changed.
```

Comment:
If you reboot the server, you will get the new network configuration as well. Can't we just make this a single step?

Reply:
ok, I simplified it.